extensions/notebook/src/protocol/notebookUriHandler.ts

```diff
@@ -70,14 +69,15 @@ export class NotebookUriHandler implements vscode.UriHandler {
 			}
 
 			let contents = await this.download(url);
-			let untitledUri = this.getUntitledUri(path.basename(uri.fsPath));
+			let untitledUriPath = this.getUntitledUriPath(path.basename(uri.fsPath));
+			let untitledUri = uri.with({ authority: '', scheme: 'untitled', path: untitledUriPath });
```

Comment:
Why are you changing this? You don't seem to use `untitledUri` anywhere now

Reply:
Ah whoops, guess I didn't expand it or something.

I'm still not understanding why this is necessary though. What happens if you leave it as it was?

Reply:
If I don't do anything then it won't open the new notebook since the authority needs to be undefined for the URI.

Reply:
That error isn't really telling me what's going on - you're saying you got that error when passing in the uri object like the code was doing before? Why would that be? The only thing that should be changing would be the parameters being added so I'm not sure why that would break this...

Reply:
I think the best way to answer most of your questions would be to show you offline. It will be clearly visible through the debugging experience.

Reply:
Discussed this as well with @Charles-Gagnon, for now, this can be handled here, but a suggestion would be to move it to the handleUri function.

Reply:
@VasuBhog I wasn't talking about this logic - this stuff is fine to keep here. It's the other code to parse out the notebook URL + parameters that I was saying I'd do in the handleUri function.